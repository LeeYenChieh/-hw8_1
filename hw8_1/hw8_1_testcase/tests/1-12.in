lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 it1cl6KYu2DjVAUpNeXZiiJFUq
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 3PjDPrdgMIx1KFlRtll29dAEPchjstPXGeqR0gCkWSYYUNoQGhtc
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 add scrollBar scrollBar scrollBar
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 add thickBlackBorder thickBlackBorder
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 add scrollBar thickBlackBorder thickBlackBorder
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 add scrollBar scrollBar
lqXytWFMmrZOqd4YzP2MFMuLznS0sqCc0JSM2KMJMIbn62sZ3 display
