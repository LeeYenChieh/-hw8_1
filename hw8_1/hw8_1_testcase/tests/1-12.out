it1cl6KYu2DjVAUpNeXZiiJFUq 
it1cl6KYu2DjVAUpNeXZiiJFUq 
it1cl6KYu2DjVAUpNeXZiiJFUq 
3PjDPrdgMIx1KFlRtll29dAEPchjstPXGeqR0gCkWSYYUNoQGhtc scrollBar scrollBar scrollBar thickBlackBorder thickBlackBorder scrollBar thickBlackBorder thickBlackBorder 
3PjDPrdgMIx1KFlRtll29dAEPchjstPXGeqR0gCkWSYYUNoQGhtc scrollBar scrollBar scrollBar thickBlackBorder thickBlackBorder scrollBar thickBlackBorder thickBlackBorder 
3PjDPrdgMIx1KFlRtll29dAEPchjstPXGeqR0gCkWSYYUNoQGhtc scrollBar scrollBar scrollBar thickBlackBorder thickBlackBorder scrollBar thickBlackBorder thickBlackBorder scrollBar scrollBar 
